yaml-tests/src/test/resources/showcasing-tests.yamsql

```diff
@@ -39,7 +39,8 @@
 options:
     # supported_version specifies what versions this works with
     # Any version greater than the provided version is expected to work
-    # When developing a new feature use !current_version to specify that it only works with the current version
+    # When developing a new feature use "!current_version" (but without the quotes) to specify that it only works with
+    # the current version
```

Comment:
Maybe add "'!current_version` will be replaced by the build system once a new release is being built"?
Also maybe "tests will be skipped when a version of the server exists that is not supported"?